tests/Lykke.Bil2.Client.BlocksReader.Tests/Configuration/LaunchSettingsFixture.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Lykke.Bil2.Client.BlocksReader.Tests.Configuration
+{
+    //https://stackoverflow.com/questions/43927955/should-getenvironmentvariable-work-in-xunit-test
+    public class LaunchSettingsFixture : IDisposable
+    {
+        public LaunchSettingsFixture()
+        {
+            try
+            {
+                using (var file = File.OpenText("Properties\\launchSettings.json"))
```

Comment:
![image](https://user-images.githubusercontent.com/14955144/53942650-b971d400-40d4-11e9-9a33-909e1d0e5c53.png)

Not adequate key names when using Microsoft.Extensions.Configration.
launchSettings is ised to avoid settings leak in github.